trm1/Era_uma_vez_uma_Lotus_Negra.html

```diff
@@ -9,6 +9,7 @@
 <body>
 <div class="background"></div>
 <div class="txtdiv"><pre id="artpre">
+<a href="Era_uma_vez_uma_Lotus_negra.txt">.txt</a>
```

Comment:
Se for dar merge desse PR e do #1, precisa lembrar de corrigir esse link.

```suggestion
<a href="Era_uma_vez_uma_Lotus_Negra.txt">.txt</a>
```